### PR TITLE
Ls: Add single file and multiple path support

### PR DESCRIPTION
**Pull request by faissaloo**
Still needs some clean up & support for list mode

### CODE REVIEW

File: Userland/ls.cpp [hunk 1]
```diff
@@ -193,8 +206,15 @@ int do_dir_short(const char* path)
...
     DIR* dirp = opendir(path);
     if (!dirp) {
-        perror("opendir");
-        return 1;
+        if (errno == ENOTDIR) {
+          int nprinted;
+          print_filesystem_object_short(path, path, &nprinted);
+          printf("\n");
+          return 0;
+        } else {
```
Comment (awesomekling) [inline Userland/ls.cpp]: No need to `else` after return. Also please use 4 spaces for indentation.

File: Userland/ls.cpp [hunk 2]
```diff
@@ -209,19 +229,16 @@ int do_dir_short(const char* path)
...
         sprintf(pathbuf, "%s/%s", path, name.characters());
-        int rc = lstat(pathbuf, &st);
-        if (rc == -1) {
-            printf("lstat(%s) failed: %s\n", pathbuf, strerror(errno));
-            return 2;
-        }
 
-        int nprinted = print_name(st, name.characters());
+        if (print_filesystem_object_short(pathbuf, name.characters(), &nprinted) == 2) {
+          return 2;
```
Comment (awesomekling) [inline Userland/ls.cpp]: Please use 4 spaces for indentation.

**Comment by awesomekling** [comment]
Oh duh, I didn't realize this was a draft PR :)

**Comment by awesomekling** [comment]
Is this ready for review?

**Comment by faissaloo** [comment]
This is ready for review now, just to be clear this isn't POSIX compliant, I don't know how you feel about POSIX compliancy, I just wrote it so it did the job.

**Comment by awesomekling** [comment]
It would be nice to achieve POSIX compliance in the long term, but let's be realistic and make small improvements as we go :)

**Comment by awesomekling** [comment]
Merged, thank you for working on this :)

**Pull request merged**